CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Minor: Update `CustomFieldInputNumber` to accept error and helpText props
```

Comment:
This is actually a breaking change: our component API requires bigmaven changes, so this is breaking. 😄 

Also, the text should probably change to `to accept errorText prop, remove error and helpText` or something like that.